providers/amazon/tests/system/amazon/aws/example_google_api_youtube_to_s3.py

```diff
@@ -73,28 +72,34 @@
 YOUTUBE_VIDEO_FIELDS = "items(id,snippet(description,publishedAt,tags,title))"
 
 SECRET_ARN_KEY = "SECRET_ARN"
+API_SERVER_URL_KEY = "ECS_example_google_api_youtube_to_s3_ExecutionApiServerUrl"
```

Comment:
I am not sure this is valid and I wonder how it could work. We might want to talk about it with @o-nikolas but I think we should:
- save the execution api endpoint in the same SSM parameter used to stored system test external parameters. Use the same key across all system test, e.g. `API_SERVER_URL`
- Update this line to `API_SERVER_URL_KEY = "API_SERVER_URL"`

Reply:
Yupp, you're correct, this will not work. I assume it was working because of the fallback to localhost.

Reply:
Yep, I incorrectly assumed how `SystemTestContextBuilder` fetched from SSM but after our discussion offline, I've removed setting the URL as a param to the function and will just be using API configuration's `base_url`

Reply:
Interesting, you sure yyou tested with this change: #49599? That's literally the goal of this PR :) To automatically include a token in the request if this config is on

Reply:
Much better :)